src/main/java/ch/njol/skript/lang/Variable.java

```diff
@@ -135,6 +136,11 @@ public static boolean isValidVariableName(String name, final boolean allowListVa
 			if (printErrors)
 				Skript.error("A variable's name must not contain the separator '" + SEPARATOR + "' multiple times in a row (error in variable {" + name + "})");
 			return false;
+		} else if (org.apache.commons.lang3.StringUtils.remove(name, SEPARATOR).contains(SINGLE_SEPARATOR_CHAR)) {
+			if (printErrors)
```

Comment:
Maybe instead: If you meant to make this variable a list, it should contain two double colons (::); currently, it contains only one.

Reply:
that's getting there. i think it's still needs some refinement tho